app/journey/Journey.scala

```diff
@@ -164,7 +164,15 @@ final case class Journey(
 }
 
 object Journey {
-  implicit val localDateTimeFormat: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
+  final val localDateTimeReads: Reads[LocalDateTime] =
```

Comment:
was there something wrong with what `MongoJavatimeFormats.localDateTimeFormat` was doing?

Reply:
MongoJavatimeFormats.localDateTimeFormat wasn't being used, now removed.
If you are talking about the json conversion, it was removed from https://github.com/hmrc/hmrc-mongo, see the readme for Version 1.0.0. I've just found the removed code and added it here to make it work again. If it's been moved/available in a different library please say what it is and I'll include that instead.

Reply:
ah ok - cool 👍